merkle/merkle_persistency.go

```diff
@@ -0,0 +1,157 @@
+package merkle
+
+import (
```

Comment:
When mention the original author of the code? :P

Reply:
Sorry for that!